runtime/common/xwalk_resource_delegate.cc

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) 2016 Intel Corporation. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "xwalk/runtime/common/xwalk_resource_delegate.h"
+
+#include "base/files/file_path.h"
+#include "base/path_service.h"
+#include "ui/gfx/image/image.h"
+
+namespace xwalk {
+
+namespace {
+
+XWalkResourceDelegate* g_instance = NULL;
```

Comment:
@wuhengzhi   I found in xwalk/runtime/  the usage of "nullptr" and "NULL" is half and half.  Is there  any policy to follow when to use nullptr and NULL?
It is a good point, thanks for pointing out this.